src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientView/PatientView.style.scss

```diff
@@ -10,57 +10,57 @@
 
     .ant-card-head {
         background-color: $elementHeader !important;
+        border: 1px solid $elementHeader !important;
+        border-color: $elementHeader !important;
         display: flex;
-        justify-content: center;
+        justify-content: left;
 
         h4 {
             margin-bottom: 0;
+            font-size: 18px;
         }
     }
     .info-container {
         display: flex;
-        justify-content: space-between;
-        span {
-            font-weight: bold;
-        }
+        justify-content: left;
         h4 {
-            font-style: italic;
             font-weight: 150;
-            font-size: 1.25rem;
+            font-size: 14px;
             margin-bottom: 0;
             margin-top: 0.8125rem;
         }
         .info-left {
+            padding-right: 100px;
             p {
-                font-weight: bold;
-                margin-left: 1.6rem;
-                font-size: 1.25rem;
+                font-size: 1rem;
+                font-weight: 500;
             }
```

Comment:
I there a reason we are not consistently using either rem or px for font size?

Reply:
We are using rem consequently. Forgot to change these fields before pushing this. Will change